tests/ast/run-unparser.sh

```diff
@@ -1,2 +1,2 @@
 #!/bin/bash
```

Comment:
I think @sfc-gh-azwiegincew recently updated this, pls reflect!

Reply:
Yes, this file is now deleted.

Reply:
pls delete this file